docs/griptape-framework/structures/tasks.md

```diff
@@ -391,3 +391,46 @@ agent.run("Give me information about Griptape")
                              flexibility when working with large language       
                              models.
 ```
+
+## Code Execution Task
+
+To execute an arbitrary python function, use the [CodeExecutionTask](../../reference/griptape/tasks/text_query_task.md).
+This task takes a python function, and authors can elect to return a custom artifact.
+
+```python 
+from griptape.structures import Pipeline
+from griptape.tasks import CodeExecutionTask, PromptTask
+from griptape.artifacts import BaseArtifact, TextArtifact
+
+
+def character_counter(task: CodeExecutionTask) -> BaseArtifact:
+    result = len(task.input)
+    # Return task.input for non-outputting functions
```

Comment:
This comment isn't accurate for this example, so remove it.

Reply:
oh or maybe I was mis-reading it. You might couch this as "for functions that don't need to return anything, we recommend returning task.input".